frugalos_config/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "frugalos_config"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Should we change the minor version of modules after re-applying rustfmt?

Reply:
In my opinion, it seems unnecessary.